content/docs/introduction/regions.md

```diff
@@ -21,6 +21,8 @@ Neon supports project deployment in several regions. We recommended that you sel
 ## Azure regions
 
 - Azure East US 2 region (Virginia) &mdash; `azure-eastus2` (**Beta**)
+- Azure Germany West Central region &mdash; `azure-gwc` (**Beta**)
+- Azure West US 3 region (Arizona) &mdash; `azure-westus3` (**Beta**)
```

Comment:
I've added the state name but we're not actually showing them in console, e.g. it's just `East US 2`, so maybe we can remove them?

Reply:
We reference this page from the Neon API docs, where you have to specify a region_id value like aws-us-east-1, although, I have not tried the Create project API with an Azure region ID. I should test that. I assume it works.